src/releasenotes_check_workflow/releasenotes_check_args.py

```diff
@@ -0,0 +1,68 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import argparse
+import logging
+from typing import IO
+
+
+class ReleaseNotesCheckArgs:
+    manifest: IO
+    gitlogdate: str
+    addcomment: bool
+    gitissuenumber: str
+    gittoken: str
+
+    def __init__(self) -> None:
+        parser = argparse.ArgumentParser(description="Checkout an OpenSearch Bundle and check for CommitID and Release Notes")
+        parser.add_argument("manifest", type=argparse.FileType("r"), help="Manifest file.")
+        parser.add_argument(
+            "-v",
+            "--verbose",
+            help="Show more verbose output.",
+            action="store_const",
+            default=logging.INFO,
+            const=logging.DEBUG,
+            dest="logging_level",
+        )
+        parser.add_argument(
+            "--gitlogdate",
+            type=str,
+            required=True,
+            dest="gitlogdate",
+            help="Date to retrive the commit (in format yyyy-mm-dd, example 2022-07-26)."
+        )
+        parser.add_argument(
+            "--addcomment",
```

Comment:
The workflow outputs the markdown table, it's not always true that we need to add comment to the GH issue, we can even view the table as output on the console (local or jenkins), once `--addcomment` is true, then it will proceed to add a comment. Did I answer your query?

Reply:
I think I explained myself poorly. 

I think that the feature of adding a comment to a GitHub issue does not belong in python code. It can be done in a 1-liner in the Jenkins job: pipe the output from the python code to a file, then call `gh` with the right arguments.

Reply:
The idea here to make this also work without jenkins as well dB, a user running on local can pass `--addcomment` and followed by the supported git args of choice which should be able to add comment to the desired repo and issue.

Reply:
Why? What human is going to run this? I can copy paste the output in an issue if I need to, this is trying to do too much.

Reply:
All our workflows has the ability to use with automation plus execute on local, with this even without jenkins we should be able to add comment to the desired issue (Like run as Job, manual GH workflow trigger, there are many ways we can use this). Hence I would recommend we have this feature `--addcomment`, to either comment on the GH issue or just output to the local.

Reply:
I have several objections to this: the biggest one being that the user is asked to supply a GitHub token, aka different from what they are already authenticated as. 

There's a lot of things going on in this PR. How about removing this functionality from this PR for now, and then doing whatever the right thing is on top of it?

Reply:
Hey @dblock Github api (using `gh` cli) uses token.
Not multiple things dB, workflow functionality is straight forward, create an info markdown table if  `--addcomment` (true) comment the same markdown table to the gh issue.

Reply:
Removing the `--addcomment` args from this workflow :)